Hud/Sprites/SpriteSheetMetadata.cs

```diff
@@ -2,19 +2,34 @@
 using OpenTK.Mathematics;
 
 namespace Hud.Sprites;
-
+/// <summary>
+/// Represents a sprite sheet metadata.
+/// </summary>
 [Serializable]
 public class SpriteSheetMetadata
 {
+    /// <summary>
+    /// Width of the sprite sheet.
+    /// </summary>
     [JsonPropertyName("width")]
     public int Width { get; set; }
```

Comment:
The properties in this class could be private fields, but I'm not sure if that would work with the serializer. If it'd be possible to make the private we could do that I reckon

Reply:
It fails with the serializer